src/Shared/PlatformSupport/AssetLoader.cs

```diff
@@ -67,7 +67,20 @@ public bool Exists(Uri uri, Uri baseUri = null)
         /// <exception cref="FileNotFoundException">
         /// The resource was not found.
         /// </exception>
-        public Stream Open(Uri uri, Uri baseUri = null)
+        public Stream Open(Uri uri, Uri baseUri = null) => OpenWithAssembly(uri, baseUri).Item2;
+        
+        /// <summary>
+        /// Opens the resource with the requested URI.
+        /// </summary>
+        /// <param name="uri">The URI.</param>
+        /// <param name="baseUri">
+        /// A base URI to use if <paramref name="uri"/> is relative.
+        /// </param>
+        /// <returns>An assembly (optional) and a stream containing the resource contents.</returns>
+        /// <exception cref="FileNotFoundException">
+        /// The resource was not found.
+        /// </exception>
+        public Tuple<Assembly, Stream> OpenWithAssembly(Uri uri, Uri baseUri = null)
```

Comment:
Also should we be using `Tuple<>`? `()` value tuples are more modern and you can name the members. I think the problem that we were having with them has gone away.

Reply:
ValueTuple issue with .NET 4.7.1 is still there and there is no workaround.

Reply:
Isn't it fixed? https://github.com/dotnet/standard/issues/567#issuecomment-356500745